pypeit/scripts/coadd_2dspec.py

```diff
@@ -31,17 +13,9 @@ def get_parser(cls, width=None):
         parser = super().get_parser(description='Coadd 2D spectra produced by PypeIt',
                                     width=width)
 
-        parser.add_argument("--file", type=str, default=None, help="File to guide 2d coadds")
-        parser.add_argument('--det', default=None, type=str,
-                            help="1-indexed detector or list of detectors that the user wants to"
-                                 "coadd. If None, all the detectors are coadded. If the spec2d are"
-                                 "mosaiced and the user wants to restrict the coadd to only selected"
-                                 "mosaics, use the parameter detnum in the coadd2d file as done in"
-                                 "run_pypeit")
-        parser.add_argument("--obj", type=str, default=None,
-                            help="Object name in lieu of extension, e.g if the spec2d files are "
-                                 "named 'spec2d_J1234+5678_GNIRS_2017Mar31T085412.181.fits' then "
-                                 "use --obj J1234+5678")
+        parser.add_argument('coadd2d_file', type=str, default=None,
```

Comment:
I want the --obj option preservers, which means --file needs to also be optional.